### ISSUE TITLE
The "Create documentation issue" should add more context to generated issue

### ISSUE THREAD

**Issue by chalin** [open-telemetry/opentelemetry.io#3934]
> As a minor, related point: I created this issue by clicking on the "Create documentation issue" link within the documentation. I assumed (wrongly) that it would automatically provide you the context/page link when I did that. Probably others will make the same assumption. Thanks, Oliver.

Thanks for calling this out, GitHub allows to send some properties with a new issue link, maybe that can be added to those links, cc @chalin

_Originally posted by @svrnm in https://github.com/open-telemetry/opentelemetry.io/issues/3929#issuecomment-1924585621_

**Comment by theletterf** [comment]
Something like this?

```
https://github.com/open-telemetry/opentelemetry.io/issues/new?assignees=&labels=bug&projects=&template=DOCS_UPDATE.md&title=Issue in {{ .Path }}{{ .Filename }}
```

https://gohugo.io/methods/page/file/

**Comment by chalin** [comment]
Yes, something like that. I know that I implemented something like this for one of the projects, let me dig that up.

**Comment by theletterf** [comment]
I'd give this one a try if you want!

**Comment by chalin** [comment]
Through a local layout override or through the Docsy repo?

You can surely give it a try though there's work planned (https://github.com/google/docsy/issues/1841) for the next release of Docsy that should make this issue _much_ easier to address -- so it might be more effective to wait until then.
